pkg/operator/nsfinalizercontroller/finalizer_controller.go

```diff
@@ -13,15 +13,15 @@ import (
 	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 	"k8s.io/apimachinery/pkg/util/wait"
 	kubeinformers "k8s.io/client-go/informers"
-	"k8s.io/client-go/kubernetes/typed/core/v1"
+	v1 "k8s.io/client-go/kubernetes/typed/core/v1"
 	appsv1lister "k8s.io/client-go/listers/apps/v1"
 	corev1listers "k8s.io/client-go/listers/core/v1"
 	"k8s.io/client-go/tools/cache"
 	"k8s.io/client-go/util/workqueue"
 	"k8s.io/klog"
 )
 
-type finalizerController struct {
+type FinalizerController struct {
```

Comment:
this change looks wrong

Reply:
would
```
interface controller {
   Run(workers int, stopChan <-)
}

struct APIServerControllerSet {
    controllers []controller
    revisionController
    encryptionControllers
}
```
be better, in order to be able to run all the controllers in the `Run()` method of the struct?

Reply:
ok, I can accept this for now.